src/sentry/event_manager.py

```diff
@@ -836,14 +836,6 @@ def save(self, project_id, raw=False, assume_normalized=False):
                 environment=environment,
             )
 
-            # Update any event attachment that arrived before the event group was defined.
-            EventAttachment.objects.filter(
-                project_id=project.id,
-                event_id=event_id,
-            ).update(
-                group_id=group.id,
```

Comment:
Isn't this used to delete dangling attachments later?

Reply:
Since (for now at least) deleting the group deletes the event (https://github.com/getsentry/sentry/blob/master/src/sentry/deletions/defaults/group.py#L36) and deleting the event removes the attachment (https://github.com/getsentry/sentry/blob/master/src/sentry/deletions/defaults/event.py#L25), I think this would still get cleaned up.

Reply:
Perhaps the situation was where we didn't receive the event at all, only the attachment.

Reply:
If we receive an attachment and no event at all doesn't that mean this code won't run? And we'll never populate a group ID?